<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLightsWidget</class>
 <widget class="QWidget" name="QgsLightsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListView" name="mLightsListView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="btnAddLight">
         <property name="text">
          <string>...</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnRemoveLight">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="mStackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_2"/>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="labelPointLightCrs">
         <property name="text">
          <string>Coordinates in 3D map CRS</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsDoubleSpinBox" name="spinPositionX">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-99999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>99999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QgsDoubleSpinBox" name="spinIntensity">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QgsDoubleSpinBox" name="spinPositionY">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-99999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>99999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QgsDoubleSpinBox" name="spinPositionZ">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-99999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>99999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QgsColorButton" name="btnColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QgsDoubleSpinBox" name="spinA0">
         <property name="maximum">
          <double>9.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QgsDoubleSpinBox" name="spinA1">
         <property name="maximum">
          <double>9.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QgsDoubleSpinBox" name="spinA2">
         <property name="maximum">
          <double>9.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Intensity</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Attenuation (A&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;+A&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&amp;times;D+A&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&amp;times;D&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="7" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Intensity</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="directionLightLayout">
         <property name="topMargin">
          <number>11</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Azimuth</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="spinBoxAzimuth">
           <property name="suffix">
            <string>Â°</string>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="directionCmpassWidget" native="true"/>
         </item>
         <item>
          <widget class="QDial" name="dialAzimuth">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="sliderPosition">
            <number>0</number>
           </property>
           <property name="wrapping">
            <bool>true</bool>
           </property>
           <property name="notchTarget">
            <double>10.000000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="altitudeSpinBox">
           <property name="text">
            <string>Altitude</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="spinBoxAltitude">
           <property name="suffix">
            <string>Â°</string>
           </property>
           <property name="minimum">
            <double>-90.000000000000000</double>
           </property>
           <property name="maximum">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderAltitude">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>90</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="7" column="1">
        <widget class="QgsDoubleSpinBox" name="spinDirectionalIntensity">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QgsColorButton" name="btnDirectionalColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Light Direction</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="labelY">
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelX">
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="labelZ">
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mLightsListView</tabstop>
  <tabstop>btnAddLight</tabstop>
  <tabstop>btnRemoveLight</tabstop>
  <tabstop>spinPositionX</tabstop>
  <tabstop>spinPositionY</tabstop>
  <tabstop>spinPositionZ</tabstop>
  <tabstop>btnColor</tabstop>
  <tabstop>spinIntensity</tabstop>
  <tabstop>spinA0</tabstop>
  <tabstop>spinA1</tabstop>
  <tabstop>spinA2</tabstop>
  <tabstop>dialAzimuth</tabstop>
  <tabstop>spinBoxAzimuth</tabstop>
  <tabstop>spinBoxAltitude</tabstop>
  <tabstop>sliderAltitude</tabstop>
  <tabstop>btnDirectionalColor</tabstop>
  <tabstop>spinDirectionalIntensity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
